Week-4/4.2/index-4.2.js

```diff
@@ -0,0 +1,48 @@
+function Person(name, age) {
+    this.name = name;
+    this.age = age;
+}
+
+Person.prototype.printDetails = function() {
```

Comment:
@mr-rego i think the method name is initialize accepting some arguments. Can you please check. But the implementation looks good to me.